gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -822,6 +822,54 @@ public static AddressFilter notEquals(AddressField field, String value) {
   }
 
   /**
+   * Class for filtering snapshot lists.
+   */
+  class SnapshotFilter extends ListFilter {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mziccard/gcloud-java/file/2301651842/issues/source?bid=3140968&fileBranchId=3194371#l827)